apps/expo/package.json

```diff
@@ -13,16 +13,16 @@
   "dependencies": {
     "@acme/api": "*",
     "@shopify/flash-list": "^1.2.1",
-    "@tanstack/react-query": "^4.7.2",
-    "@trpc/client": "10.0.0-proxy-beta.13",
-    "@trpc/react": "10.0.0-proxy-beta.13",
-    "@trpc/server": "10.0.0-proxy-beta.13",
+    "@tanstack/react-query": "^4.14.1",
+    "@trpc/client": "10.0.0-rc.3",
+    "@trpc/react-query": "10.0.0-rc.3",
```

Comment:
Hey, noticed that the trpc util for expo hasn't been updated to match this change
https://github.com/t3-oss/create-t3-turbo/blob/f30c8af893f11f0145c92b4845dd9bb14cfe4df4/apps/expo/src/utils/trpc.tsx#L1